tslint.json

```diff
@@ -1,14 +1,55 @@
 {
-    "rules": {
-        "no-any": false,
-        "no-empty": true,
-        "no-empty-interface": true,
-        "no-duplicate-imports": true,
-        "no-duplicate-variable": true,
-        "no-unused-variable": true,
-        "triple-equals": true,
-        "prefer-const": true,
-        "arrow-return-shorthand": true,
-        "no-invalid-template-strings": true
-    }
-}
\ No newline at end of file
+  "extends": ["tslint:recommended", "tslint-config-prettier"],
```

Comment:
Find and remove rule which tells about namespace and modules invalid usage - we using namespaces almost in all React files